ipamutils/utils.go

```diff
@@ -13,11 +14,22 @@ var (
 	// PredefinedGranularNetworks contains a list of 64K IPv4 private networks with host size 8
 	// (10.x.x.x/24) which do not overlap with the networks in `PredefinedBroadNetworks`
 	PredefinedGranularNetworks []*net.IPNet
-
-	initNetworksOnce sync.Once
+	initNetworksOnce           sync.Once
```

Comment:
kind of confuse on the mechanics of this initNetworksOnce, I see it used into 2 points.
1) line 34
2) line 93

Where can I find the code that is using the new method?

Reply:
@selansen can you reply to this question?

Reply:
Has this been clarified ?